doc/book/container/aura-di.md

```diff
@@ -0,0 +1,118 @@
+# Using Aura.Di
+
+[Aura.Di](https://github.com/auraphp/Aura.Di/) provides a serializable dependency
+injection container with the following features:
+
+- constructor and setter injection
+
+- inheritance of constructor parameter and setter method values from parent classes
+
+- inheritance of setter method values from interfaces and traits
+
+- lazy-loaded instances, services, includes/requires, and values
+
+- instance factories
+
+- optional auto-resolution of typehinted constructor parameter values
+
+
+## Installing Aura.Di
+
+Aura.Di v3 only implements [container-interop](https://github.com/container-interop/container-interop).
+
+```bash
+$ composer require "aura/di:3.0.*@beta"
+```
+
+## Configuration
+
+Aura.Di can help you to reorganize your code better with
+[ContainerConfig classes](http://auraphp.com/packages/Aura.Di/config.html) and
+[two step configuration](http://auraphp.com/blog/2014/04/07/two-stage-config/)
+in this example, we'll have that in `config/services.php`.
+
+```php
+<?php
+use Aura\Di\ContainerBuilder;
+
+$container_builder = new ContainerBuilder();
+
+// use the builder to create and configure a container
+// using an array of ContainerConfig classes
+// make sure the classes can be autoloaded
+return $container_builder->newConfiguredInstance([
+    'Application\_Config\Common',
+]);
+```
+
+The bare minimal `ContainerConfig ` code needed to make zend-expressive work is
+
+```php
```

Comment:
Normally in `config` , and add autoload to composer.json . See skelton https://github.com/harikt/expressive-aura-skelton/blob/bbbbf233170ee4b76877860d676116f44f1d75d2/composer.json#L11 .